obsreport/obsreport_receiver.go

```diff
@@ -68,19 +68,39 @@ var (
 		stats.UnitDimensionless)
 )
 
+// StartReceiveOptions has the options related to starting a receive operation.
+type StartReceiveOptions struct {
+	// LongLivedCtx when true indicates that the context passed in the call
+	// outlives the individual receive operation.
```

Comment:
Can you add a bit more explanation of what "outlives" means and what is an "individual receive operation"? Generally it would be good to see better explanation of long-lived context concept.